README.md

```diff
@@ -41,12 +51,14 @@ We meet [Collection Requirements](https://github.com/ansible-collections/overvie
 ```
 In case the repository is not on GitHub, please provide email addresses.
 
-2. Ensure any other maintainers of the collection are `@mentioned` in the discussion so they will receive updates.
-3. Promptly reply to any questions.
+If you reviewed someone else's collection, please specify it too.
+
+3. Ensure any other maintainers of the collection are `@mentioned` in the discussion so they will receive updates.
+4. Promptly reply to any questions.
 
 ## Review process
 
-1. Choose a review request from the [New collection reviews](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/new-collection-reviews) or [Second review needed](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/second-review-needed) categories.
+1. Choose a review request from the [New collection reviews](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/new-collection-reviews) or [Second review needed](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/second-review-needed) categories. Give a much higher priority to collections whose maintainers reviewed someone else's collection (see discussion descriptions for this information).
```

Comment:
```suggestion
1. Choose a review request from the [New collection reviews](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/new-collection-reviews) or [Second review needed](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/second-review-needed) categories. Give a higher priority to collections whose maintainers reviewed someone else's collection (see discussion descriptions for this information).
```

I would remove the `much` here. This might raise expectations we're not ready to live up to.

Reply:
Agreed, committed, thanks!